WordPressVIPMinimum/Tests/VIP/PHPFilterFunctionsUnitTest.inc

```diff
@@ -0,0 +1,24 @@
+<?php
+
+$url = 'http://www.google.ca';
+$_GET['foo'] = 'bar';
+$array = [ 'something_something', 'https://www.google.com', '6' ];
+
+// Ok.
+filter_var( $url, FILTER_SANITIZE_URL );
+filter_var( 'test', FILTER_SANITIZE_STRING );
+filter_var(  "test", FILTER_SANITIZE_STRING );
+filter_input( INPUT_GET, 'foo', FILTER_SANITIZE_STRING );
+filter_var_array( $array, FILTER_SANITIZE_STRING );
+filter_input_array( $array, FILTER_SANITIZE_STRING );
+
```

Comment:
Maybe add another one or two with some non-WPCS whitespace arrangements, to show that logic in the Sniff is whitespace-agnostic.